packages/elvis/src/components/form/radio-panel.scss

```diff
@@ -0,0 +1,222 @@
+@use '../../utilities/breakpoints';
+@use '../../utilities/mixins';
+@use '../../utilities/outline';
+
+@mixin limit-lines($line-clamp) {
+  overflow: hidden;
+  display: -webkit-box;
+  -webkit-line-clamp: $line-clamp;
+  -webkit-box-orient: vertical;
+}
+
+@mixin square($size) {
+  height: $size;
+  width: $size;
+}
+
+.e-radio-panel-container {
+  --min-width: 150px;
+
+  display: grid;
+  gap: 8px;
+  grid-auto-rows: 1fr; //same height rows
+  place-items: stretch;
+  grid-template-columns: repeat(auto-fill, minmax(var(--min-width), 1fr));
```

Comment:
I'm not satisfied with this layout as it often makes the radio panels smaller than expected. It uses the same strategic as all cars on our documentation page. However, autofill works better than auto-fit, as auto-fit often breaks when the screen gets too wide, or if there are only a few panels. 😕

It currently works like this:

https://github.com/user-attachments/assets/b245983a-03a3-4c7f-887f-8cb550ffd720